vaultvisor/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "vaultvisor"
```

Comment:
Tempted to suggested we name this project `vault-runner` or simply `runner` for clarity.